tests/framework/framework.go

```diff
@@ -135,7 +135,7 @@ type Framework struct {
 // cannot work when called during test tree construction.
 func NewFramework(prefix string, config ...Config) *Framework {
 	cfg := Config{
-		FeatureGates: []string{featuregates.HonorWaitForFirstConsumer},
+		FeatureGates: []string{featuregates.HonorWaitForFirstConsumer, featuregates.WebhookPvcRendering},
```

Comment:
Could we also make the previously related Serial tests now parallel?

Reply:
Yep, fixing that.